### PR TITLE
Update MS-ASCAL RS and XML

### PR DESCRIPTION
**Pull request by XINYU33**

### CODE REVIEW

File: ExchangeActiveSync/Source/MS-ASCAL/TestSuite/MS-ASCAL_RequirementSpecification.xml [hunk 1]
```diff
@@ -1864,6 +1963,15 @@ AirSync [and reference is] [MS-ASCMD] section 2.2.2.19.</ns1:Description>
 		<ns1:IsNormative>Normative</ns1:IsNormative>
 		<ns1:Verification>Adapter</ns1:Verification>
 	</ns1:Requirement>
+	<ns1:Requirement>
+		<ns1:REQ_ID>MS-ASCAL_R2235001</ns1:REQ_ID>
+		<ns1:Doc_Sect>2.2.2.9</ns1:Doc_Sect>
+		<ns1:Description>[In BusyStatus] The value -1 (Status Unknow) is not supported in protocol versions 2.5, 12.0, 12.1[, 14.0, and 14.1].</ns1:Description>
```
Comment (XinwLi) [inline ExchangeActiveSync/Source/MS-ASCAL/TestSuite/MS-ASCAL_RequirementSpecification.xml]: each protocol version should drive one RS.

File: ExchangeActiveSync/Source/MS-ASCAL/TestSuite/MS-ASCAL_RequirementSpecification.xml [hunk 2]
```diff
@@ -6132,6 +6546,15 @@ AirSync [and reference is] [MS-ASCMD] section 2.2.2.19.</ns1:Description>
...
 		<ns1:Verification>Test Case</ns1:Verification>
 	</ns1:Requirement>
+	<ns1:Requirement>
+		<ns1:REQ_ID>MS-ASCAL_R2175001</ns1:REQ_ID>
+		<ns1:Doc_Sect>2.2.2.36</ns1:Doc_Sect>
+		<ns1:Description>[In OrganizerName] [When protocol version 16.1 is used, the client MUST NOT include the OrganizerName element in command requests and] the server will use the name of the current user.</ns1:Description>
+		<ns1:Behavior>Protocol</ns1:Behavior>
+		<ns1:Scope>Server</ns1:Scope>
+		<ns1:IsNormative>Normative</ns1:IsNormative>
+		<ns1:Verification>Test Case</ns1:Verification>
```
Comment (XinwLi) [inline ExchangeActiveSync/Source/MS-ASCAL/TestSuite/MS-ASCAL_RequirementSpecification.xml]: R2175001 is adapter

File: ExchangeActiveSync/Source/MS-ASCAL/TestSuite/MS-ASCAL_RequirementSpecification.xml [hunk 3]
```diff
@@ -6162,12 +6585,39 @@ AirSync [and reference is] [MS-ASCMD] section 2.2.2.19.</ns1:Description>
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-ASCAL_R374</ns1:REQ_ID>
```
Comment (XinwLi) [inline ExchangeActiveSync/Source/MS-ASCAL/TestSuite/MS-ASCAL_RequirementSpecification.xml:6586]: merge R374, R374001 and R374002 to one RS is better.

File: ExchangeActiveSync/Source/MS-ASCAL/TestSuite/MS-ASCAL_RequirementSpecification.xml [hunk 4]
```diff
@@ -6162,12 +6585,39 @@ AirSync [and reference is] [MS-ASCMD] section 2.2.2.19.</ns1:Description>
...
+		<ns1:IsNormative>Normative</ns1:IsNormative>
+		<ns1:Verification>Adapter</ns1:Verification>
+	</ns1:Requirement>
+	<ns1:Requirement>
+		<ns1:REQ_ID>MS-ASCAL_R374003</ns1:REQ_ID>
+		<ns1:Doc_Sect>2.2.2.37</ns1:Doc_Sect>
+		<ns1:Description>[In Recurrence] [The Recurrence element can have the following child elements:]Type (section 2.2.2.45): optional in protocol versions 16.0 and 16.1.</ns1:Description>
+		<ns1:Behavior>Protocol</ns1:Behavior>
+		<ns1:Scope>Both</ns1:Scope>
+		<ns1:IsNormative>Informative</ns1:IsNormative>
```
Comment (XinwLi) [inline ExchangeActiveSync/Source/MS-ASCAL/TestSuite/MS-ASCAL_RequirementSpecification.xml]: Normative

File: ExchangeActiveSync/Source/MS-ASCAL/TestSuite/MS-ASCAL_RequirementSpecification.xml [hunk 5]
```diff
@@ -7671,15 +8193,42 @@ AirSync [and reference is] [MS-ASCMD] section 2.2.2.19.</ns1:Description>
...
+		<ns1:Behavior>Protocol</ns1:Behavior>
+		<ns1:Scope>Both</ns1:Scope>
+		<ns1:IsNormative>Normative</ns1:IsNormative>
+		<ns1:Verification>Unverified</ns1:Verification>
+	</ns1:Requirement>
 	<ns1:Requirement>
 		<ns1:REQ_ID>MS-ASCAL_R448</ns1:REQ_ID>
 		<ns1:Doc_Sect>2.2.2.45</ns1:Doc_Sect>
-		<ns1:Description>[In Type] The Type element is a required child element of the Recurrence element (section 2.2.2.35) that specifies the type of the recurrence.</ns1:Description>
+		<ns1:Description>[In Type] The Type element is a required child element of the Recurrence element (section 2.2.2.35) in protocol version 2.5, 12.0, 12.1, 14.4, and 14.1; it is an optional child element of the Recurrence element in protocol versions 16.0 and 16.1.</ns1:Description>
```
Comment (XinwLi) [inline ExchangeActiveSync/Source/MS-ASCAL/TestSuite/MS-ASCAL_RequirementSpecification.xml]: remove "it is an optional child element of the Recurrence element in protocol versions 16.0 and 16.1" from this RS

File: ExchangeActiveSync/Source/MS-ASCAL/TestSuite/MS-ASCAL_RequirementSpecification.xml [hunk 6]
```diff
@@ -7671,15 +8193,42 @@ AirSync [and reference is] [MS-ASCMD] section 2.2.2.19.</ns1:Description>
...
 		<ns1:IsNormative>Normative</ns1:IsNormative>
 		<ns1:Verification>Adapter</ns1:Verification>
 	</ns1:Requirement>
+	<ns1:Requirement>
+		<ns1:REQ_ID>MS-ASCAL_R448001</ns1:REQ_ID>
+		<ns1:Doc_Sect>2.2.2.45</ns1:Doc_Sect>
+		<ns1:Description>[In Type] [The Type element is a required child element of the Recurrence element (section 2.2.2.35) in protocol version 2.5, 12.0, 12.1, 14.4, and 14.1; ]it is an optional child element of the Recurrence element in protocol versions 16.0 and 16.1.</ns1:Description>
+		<ns1:Behavior>Protocol</ns1:Behavior>
+		<ns1:Scope>Both</ns1:Scope>
+		<ns1:IsNormative>Informative</ns1:IsNormative>
```
Comment (XinwLi) [inline ExchangeActiveSync/Source/MS-ASCAL/TestSuite/MS-ASCAL_RequirementSpecification.xml]: normative

File: ExchangeActiveSync/Source/MS-ASCAL/TestSuite/MS-ASCAL_RequirementSpecification.xml (hunk 2 above)
Comment (XINYU33) [reply ExchangeActiveSync/Source/MS-ASCAL/TestSuite/MS-ASCAL_RequirementSpecification.xml:6556]: R2175 is verified in test case, should we update the test case R2175001 base on it?

Comment (XINYU33) [reply ExchangeActiveSync/Source/MS-ASCAL/TestSuite/MS-ASCAL_RequirementSpecification.xml]: R2175 is verified in test case, should we update the test case R2175001 base on it?

Comment (XinwLi) [reply ExchangeActiveSync/Source/MS-ASCAL/TestSuite/MS-ASCAL_RequirementSpecification.xml]: yes， R2175 should be adapter

**Review by XinwLi — APPROVED** [review]

**Pull request merged**